Makefile.work

```diff
@@ -509,9 +509,13 @@ endif
 
 ifeq "$(KEEP_SLAVE_ON)" "yes"
 SLAVE_SHELL={ /bin/bash; }
+else
+SLAVE_SHELL={ true; }
 endif
+ERROR_EXIT={ $(SLAVE_SHELL); exit -1; }
```

Comment:
Looks like it can be simplified as below:
```
ifeq "$(KEEP_SLAVE_ON)" "yes"
SLAVE_SHELL={ ret=\$$?; /bin/bash; exit $ret; }
endif
```

Reply:
Done, thanks. it is much simpler now